duckduckgo/privacy-dashboard
# ios: Increase size of drop down button

https://app.asana.com/0/0/1206073695355523/f



## Description:



## Steps to test this PR:

1. use this branch of the dashboard `12-01-ios_Increase_size_of_drop_down_button`
2. check the height on device matches the screenshot seen in the diff
